web/versioned/browsers-0.3.2.bzl

```diff
@@ -56,39 +56,39 @@ def org_chromium_chromedriver():
         name = "org_chromium_chromedriver",
         licenses = ["reciprocal"],  # BSD 3-clause, ICU, MPL 1.1, libpng (BSD/MIT-like), Academic Free License v. 2.0, BSD 2-clause, MIT
         amd64_sha256 =
-            "f6b9852031d185739a2c1816508fe8158eb92782d13e831b8345957ef2506fe8",
+            "c8b8be2fc6835bd3003c16d73b9574242e215e81e9b3e01d6fed457988d052f4",
         amd64_urls = [
-            "https://commondatastorage.googleapis.com/chromium-browser-snapshots/Linux_x64/768968/chromedriver_linux64.zip",
+            "https://commondatastorage.googleapis.com/chromium-browser-snapshots/Linux_x64/870763/chromedriver_linux64.zip",
```

Comment:
Do we have any documentation or notes talking about how we could get this 870763 number?

If so, could we add a link to that doc in the comment? If no, could we at least add a short comment to explain how to get it?

Reply:
Added comments.